tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -656,10 +661,10 @@ def sum_of_reoccurring_values(x):
     :return: the value of this feature
     :return type: float
     """
-    x = pd.Series(x)
-    value_counts = x.value_counts()
-    doubled_values = value_counts[value_counts > 1]
-    return sum(doubled_values.index * doubled_values)
+
+    unique, counts = np.unique(x, return_counts=True)
```

Comment:
Sorry my last remark was wrong. This feature really expresses the sum of the recurring values.   

Can your reverse parts of last commit?  The following lines are fine for the `sum_of_reoccurring_values` feature

```
unique, counts = np.unique(x, return_counts=True)
counts[counts < 2] = 0
return np.sum(counts * unique)
```